index.html

```diff
@@ -71,8 +70,210 @@ <h2>I'am a full-stack Web developer</h2>
+                    </div>
+                </div>
+                <div class="seemore">
+                    <a href="">See Project</a>
+                </div>
+            </div>
+            <div class="card-block">
+                <div class="card-pic">
+                    <div class="card-info">
+                        <div class="card-title">
+                            Aura Patient Record System
+                        </div>
+                        <div class="card-discription">
+                            A record system for patients info and treatment data at AURA health care clinic.
+                        </div>
+                        <div class="card-lang">
+                            <div>html</div>
+                            <div>css</div>
+                            <div>php</div>
+                            <div>laravel</div>
+                        </div>
+                    </div>
+                </div>
+                <div class="seemore">
+                    <a href="">See Project</a>
+                </div>
+            </div>
+        </div>
+    </section>
+    <!--About Me-->
+    <section class="about-me">
+        <div class="home-content">
+            <div class="heading">
+                About me
+            </div>
+            <div class="bio">
+                I am full-stack web developer. Allow me to make your business connect with many people. Hope we can work together.
+            </div>
+            <div class="resume">
+                <a href="resources/resume/Aung Myat Kyaw CV.pdf" target="">Get My Resume</a>
+            </div>
+        </div><hr>
+    </section>
+    <section class="personal-skill">
+        <div class="language">
+            <div class="name">
+                <div class="obli-square"></div>
+                <span>Language</span>
+            </div>
+            <div class="skill-title">Javascript</div>
+            <div class="skill-title">Ruby</div>
+            <div class="skill-title">Html</div>
+            <div class="skill-title">Css</div>
         </div>
-    </div> -->
+        <div class="language">
+            <div class="name">
+                <div class="square"></div>
+                <span>Frameworks</span>
+            </div>
+            <div class="skill-title">Ruby on Rails</div>
+            <div class="skill-title">Bootstrap</div>
+            <div class="skill-title">RSPec</div>
+            <div class="skill-title">CapyBara</div>
+            <div class="skill-title">Selenium</div>
+        </div>
+        <div class="language">
+            <div class="name">
+                <div class="circle"></div>
+                <span>Skill</span>
+            </div>
+            <div class="skill-title">Codekit</div>
+            <div class="skill-title">Github</div>
+            <div class="skill-title">Codepen</div>
+            <div class="skill-title">Gitlab</div>
+            <div class="skill-title">Terminal</div>
```

Comment:
- [ ] Good job creating all the required elements to match with the Figma template 👍, but for semantics, the technology categories ("Languages", "Frameworks" and "Skills") should be in a `ul`, and each of them in a `li` tag. And also the technology in each category(e.g., "JavasScript", "HTML" and "CSS") should be a list `ul` and each of them a `li` 💯